src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -2947,19 +2949,19 @@ TEST_P(conv_int8_eltwise_onednn, u8_eltwise_prod_out) {
 }
 
 INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_int8_eltwise_onednn, ::testing::ValuesIn(std::vector<convolution_test_params>{
-    convolution_test_params{ CASE_CONV_U8S8_1, 2, 4 },
-    convolution_test_params{ CASE_CONV_U8S8_2, 2, 4 },
-    convolution_test_params{ CASE_CONV_U8S8_3, 2, 4 },
-    convolution_test_params{ CASE_CONV_S8S8_1, 2, 4 },
-    convolution_test_params{ CASE_CONV_S8S8_2, 2, 4 },
-    convolution_test_params{ CASE_CONV_S8S8_3, 2, 4 },
-
-    convolution_test_params{ CASE_CONV_U8S8_11, 2, 4 },
-    convolution_test_params{ CASE_CONV_U8S8_12, 2, 4 },
-    convolution_test_params{ CASE_CONV_U8S8_13, 2, 4 },
-    convolution_test_params{ CASE_CONV_S8S8_12, 2, 4 },
-    convolution_test_params{ CASE_CONV_S8S8_13, 2, 4 },
-    convolution_test_params{ CASE_CONV_S8S8_14, 2, 4 },
+    convolution_test_params{ CASE_CONV_U8S8_1, 3, 4 },
+    convolution_test_params{ CASE_CONV_U8S8_2, 3, 4 },
+    convolution_test_params{ CASE_CONV_U8S8_3, 3, 4 },
+    convolution_test_params{ CASE_CONV_S8S8_1, 3, 4 },
+    convolution_test_params{ CASE_CONV_S8S8_2, 3, 4 },
+    convolution_test_params{ CASE_CONV_S8S8_3, 3, 4 },
+
+    convolution_test_params{ CASE_CONV_U8S8_11, 3, 4 },
+    convolution_test_params{ CASE_CONV_U8S8_12, 3, 4 },
+    convolution_test_params{ CASE_CONV_U8S8_13, 3, 4 },
+    convolution_test_params{ CASE_CONV_S8S8_12, 3, 4 },
+    convolution_test_params{ CASE_CONV_S8S8_13, 3, 4 },
+    convolution_test_params{ CASE_CONV_S8S8_14, 3, 4 },
```

Comment:
Do we need to increase this value even if binary_mul fusion is enabled now?

Reply:
binary_mul issue is for other testcase(conv_fp32_prelu_eltwise.basic_prod/0). In this test, changed value 2 to 3 because It seems that the best fusion senario of `input|conv|eltwise|crop|(reorder)` is `input|conv+eltwise|crop|(reorder)`.